dash-info.yaml

```diff
@@ -1,10 +1,2612 @@
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFieldset(
+                children = list('Choose your favorite Dash HTML component',
+                dccRadioItems(
+                  options=list(
+                    list("label'= "htmlDiv", "value'= "htmlDiv"),
+                    list("label'= "htmlBase", "value'= "htmlBase"),
+                    list("label'= "htmlArticle", "value'= "htmlArticle")
+                        )
+                      )
+                    )
+                  )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlFigcaption
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFigure(children = list(
+                htmlImg(src = 'https://brand.plot.ly/static/images/plotly-logo-01-stripe@2x.png'),
+                htmlFigcaption(children = 'Plotly Logo')))
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlFigure
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFigure(children = list(
+                htmlImg(src = 'https://brand.plot.ly/static/images/plotly-logo-01-stripe@2x.png',
+                        width = '400',
+                        height = '150')
+                    )
+                  )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlFont
+      dontrun: TRUE
+      code: |
+            # Starting with HTML 4, HTML does not convey styling information
+            # anymore (outside the <style> element or the style attribute of each
+            # element). CSS should be used for styling instead.
+    - name: htmlFooter
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFooter(list(
+                htmlH1('Dash'),
+                htmlLi('Pointer1'),
+                htmlLi('Pointer2')
+                    )
+                  )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlForm
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlForm(children=list(
+                htmlP(children=list('Username: ', dccInput(type='text', id='username', placeholder='username'))),
+                htmlP(children=list('Password: ', dccInput(type='password', id='password', placeholder='password'))),
```

Comment:
you need `library(dashCoreComponents)` for this example

Reply:
fixed in 209a10d